lambda.js

```diff
@@ -53,7 +54,12 @@ exports.handler = (event, context, handleCallback) => {
     invocationEvent = event;
     invocationContext = context;
     cleanHeaders(event);
-    const webResponse = awsServerlessExpress.proxy(server, event, context);
+    const webResponse = awsServerlessExpress.proxy(
+      server,
+      event,
+      context,
+      "PROMISE"
```

Comment:
Why are we passing a string called `"PROMISE"` here? It looks like we never use it? Is it something specific and internal to AWS?

Reply:
Got it from here:
https://github.com/awslabs/aws-serverless-express/issues/234#issuecomment-492529316
(or a similar place in aws-serverless-express documentation that I have since lost).
Basically, it has a weird API, but that's how you 'ask' for a promise-based call.

Reply:
👍